packages/odyssey-react/src/utils/omit.ts

```diff
@@ -10,16 +10,15 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
```

Comment:
can we add comments stating why are doing omitting these attributes. It will useful for the future.

Reply:
We could! I'd prefer to add that surrounding information to the test suite for now. One day this util may be published in its own package or even documented on the hosted site. Those will be good places to provide this type of info but for now I'd like to avoid the temptation to add explanatory code comments

Reply:
@rajaguruduraisamy-okta [checkout the new commit](https://github.com/okta/odyssey/pull/974/commits/56669c709c5809facbe0eb119a8444c81a0f2ba3)

Reply:
I agree with both of you. If there is a thought that this could live on the docs site at some point, it might be worth using jsdoc here. I would love to live in a world where we use our code and storybook as the SSoT for the docs site content!